zeppelin/zeppelin.sh

```diff
@@ -19,9 +19,15 @@
 
 set -euxo pipefail
 
-readonly NOT_SUPPORTED_MESSAGE="Zeppelin initialization action is not supported on Dataproc ${DATAPROC_VERSION}.
+# Detect dataproc image version from its various names
+if (! test -v DATAPROC_IMAGE_VERSION) && test -v DATAPROC_IMAGE_VERSION; then
```

Comment:
This doesn't look correct. I guess you might want:

```
if (! test -v DATAPROC_IMAGE_VERSION) && test -v DATAPROC_VERSION; then
  DATAPROC_IMAGE_VERSION="${DATAPROC_VERSION}"
fi
```

Reply:
You're right!  We were too aggressive about the search and replace.  Thanks for catching this!